apps/web/pages/auth/forgot-password/[id].tsx

```diff
@@ -159,7 +159,7 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
   });
   try {
     resetPasswordRequest &&
-      (await prisma.user.findUniqueOrThrow({ where: { email: resetPasswordRequest.email } }));
+      (await prisma.user.findFirstOrThrow({ where: { email: resetPasswordRequest.email } }));
```

Comment:
This is to verify there is a reset password request and that there is a user to apply that request, finding at least one user is enough.